Challenge-4/Bot_2/src/constants.ts

```diff
@@ -2,8 +2,16 @@ import { ethers } from "forta-agent";
 import { BotConfig } from "./types";
 
 export const CONFIGURATION_ABI = [
-  `function getConfiguration(address cometProxy) view returns (tuple(address governor, address pauseGuardian, address baseToken, address
-   baseTokenPriceFeed, address extensionDelegate, uint64 supplyKink, uint64 supplyPerYearInterestRateSlopeLow, uint64 supplyPerYearInterestRateSlopeHigh, uint64 supplyPerYearInterestRateBase, uint64 borrowKink, uint64 borrowPerYearInterestRateSlopeLow, uint64 borrowPerYearInterestRateSlopeHigh, uint64 borrowPerYearInterestRateBase, uint64 storeFrontPriceFactor, uint64 trackingIndexScale, uint64 baseTrackingSupplySpeed, uint64 baseTrackingBorrowSpeed, uint104 baseMinForRewards, uint104 baseBorrowMin, uint104 targetReserves, tuple(address asset, uint8 decimals, uint256 conversionFactor)[] assetConfigs) configuration)`,
+  `function getConfiguration(address cometProxy) view returns (tuple(address governor, address pauseGuardian, 
+   address baseToken, address baseTokenPriceFeed, address extensionDelegate, uint64 supplyKink, uint64 supplyPerYearInterestRateSlopeLow, uint64 supplyPerYearInterestRateSlopeHigh, 
+   uint64 supplyPerYearInterestRateBase, uint64 borrowKink, 
+   uint64 borrowPerYearInterestRateSlopeLow, 
+   uint64 borrowPerYearInterestRateSlopeHigh, 
+   uint64 borrowPerYearInterestRateBase, uint64 storeFrontPriceFactor, 
+   uint64 trackingIndexScale, uint64 baseTrackingSupplySpeed, 
+   uint64 baseTrackingBorrowSpeed, uint104 baseMinForRewards, 
+   uint104 baseBorrowMin, uint104 targetReserves, 
+   tuple(address asset, uint8 decimals, uint256 conversionFactor)[] assetConfigs) configuration)`,
```

Comment:
Much better, but I think for the highest legibility, you could put every value in its own line. And when it comes to the `tuple`s, those could be "opened" on one line, then list out the values on their own line, then "close" it on its own line.

For example,
```
function dummyFunction(address dummyArg) view returns (
    tuple(
        uint256 dummyValue,
        address dummyValueTwo,
        tuple(
            uint8 dummyValueThree,
            bool dummyValueFour
        ),
        uint128 dummyValueFive,
    ),
    uint64 dummyValueSix
)
```